install_config/registry/accessing_registry.adoc

```diff
@@ -272,3 +272,95 @@ cf2616975b4a: Image successfully pushed
 Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
+
+[[access-reading-and-writing-image-signatures]]
+=== Reading and Writing Image Signatures
+The {product-title} integrated Docker Registry provides an "extensions" endpoint that
+allows you to write and read image signatures. The image signatures are stored in
+{product-title} API, but this endpoint allows you to read and write them without
```

Comment:
How about, "The image signatures are stored in the {product-title} key-value store via API"?